### PR TITLE
Saq, Tf Tests and Mcq, Saq, Tf InputCommandTests 

### PR DESCRIPTION
**Pull request by jxofficial**

### CODE REVIEW

File: src/test/java/seedu/revision/logic/commands/AddCommandIntegrationTest.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import static seedu.revision.logic.commands.CommandTestUtil.assertCommandFailure;
 import static seedu.revision.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.revision.testutil.TypicalMcqs.getTypicalRevisionTool;
+import static seedu.revision.testutil.TypicalMcqs.getMcqRevisionTool;
```
Comment (wilfredbtan) [inline src/test/java/seedu/revision/logic/commands/AddCommandIntegrationTest.java]: Should refactor to getTypicalMcqs

File: src/test/java/seedu/revision/logic/parser/ListCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package seedu.revision.logic.parser;
+
+public class ListCommandParserTest {
```
Comment (wilfredbtan) [inline src/test/java/seedu/revision/logic/parser/ListCommandParserTest.java]: Is this an empty class?

File: src/test/java/seedu/revision/ui/StartQuizWindowTest.java [hunk 3]
```diff
@@ -125,7 +125,9 @@ public void typeValidAnswer_endOfLevelReached_shouldShowNextLevelAlert(FxRobot r
         robot.write(ANSWER_VALID_MCQ);
         robot.type(KeyCode.ENTER);
         int numberOfAlerts = getNumberOfWindowsShown(robot, AlertDialog.NEXT_LEVEL_TITLE);
-        assertEquals(1, numberOfAlerts);
+        assertEquals(
```
Comment (wilfredbtan) [inline src/test/java/seedu/revision/ui/StartQuizWindowTest.java]: Does this pass checkstyle? Can check the formatting in the source code.

File: src/test/java/seedu/revision/logic/parser/ListCommandParserTest.java (hunk 2 above)
Comment (jxofficial) [reply src/test/java/seedu/revision/logic/parser/ListCommandParserTest.java]: deleted

File: src/test/java/seedu/revision/ui/StartQuizWindowTest.java (hunk 3 above)
Comment (jxofficial) [reply src/test/java/seedu/revision/ui/StartQuizWindowTest.java]: Changed formatting

**Review by wilfredbtan — APPROVED** [review]

**Comment by wilfredbtan** [comment]
LGTM

**Pull request merged**